account_invoice_blocking/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```

Comment:
@Nikul-Chaudhary Please create the readme as described [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst).

Reply:
@ThomasBinsfeld README generates automatically, So not needed.

Reply:
@Nikul-Chaudhary Didn't notice they're already here :+1: